php/Terminus/Loggers/JLogger.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Terminus;
+namespace Terminus\Loggers;

```

Comment:
Cleanliness. It feels like the top level Terminus space is getting kind of crowded with a lot of very disparate types of classes.

I could also make the case for having more than one logging class. The KLogger is great at standard file-based logging, but you've had to stretch it a bit to have it log json to STDERR. At some point we might wan to clean this up a bit to have a different logger class for different modes. Minor thing though, I can move it back. I don't want to get into premature refactoring.